scripts/set_kokkos_env.sh

```diff
@@ -0,0 +1,3 @@
+export MPICH_CXX=/home/uy7/software/Trilinos/packages/kokkos/config/nvcc_wrapper
```

Comment:
I agree but the other scripts we have do not seem to be general either. These are simply examples. Should we have a policy that the examples be more general and usable with user-specified environment variables?